lib/ErrorManager.php

```diff
@@ -20,9 +20,48 @@ public function __construct(Configuration $configuration)
         $this->configuration = $configuration;
     }
 
+    public function addError(Error $error): void
```

Comment:
Might be more handy to have a method that takes scalar arguments and builds an `Error` VO internally, instead of having `addError(new Error(…))` every time

Reply:
Sure

Reply:
With this change, I wonder if I should use the existing `error`/`warning` method instead. The only downside is the argument order.. But I can live with `->error($message, $throwable, $file, $line)`

Reply:
Or you could target 0.5.x and break BC :)

Reply:
Which way will make it quicker for me to use the RST parser in my project?
 =)

Reply:
Well we just released `0.4.0`, so it wouldn't make a lot of sense to release `0.5.0` with just that sense. That being said, since `ErrorManager` is not final, adding arguments such as `$file` and `$line` would be a BC break for extending classes, so I'm not sure there is a choice either way.